prime-router/metadata/fhir_mapping/hl7/secondary/Hospitalization.yml

```diff
@@ -0,0 +1,55 @@
+preAdmissionIdentifier:
```

Comment:
When you import a copy of a config, make a comment where you made a change.  Otherwise it is hard to tell.